components/favoris/offres/OffreRow.tsx

```diff
@@ -1,30 +1,53 @@
-import React from 'react'
+import React, { useMemo } from 'react'
 
+import IconComponent, { IconName } from 'components/ui/IconComponent'
 import { Tag } from 'components/ui/Tag'
 import { Offre } from 'interfaces/favoris'
 
-export default function OffreRow({ offre }: { offre: Offre }) {
+export default function OffreRow({
+  offre,
+  handleRedirectionOffre,
+}: {
+  offre: Offre
+  handleRedirectionOffre: (offre: Offre) => void
+}) {
+  const hasLink: boolean = useMemo(
+    () => offre.hasLinkPE || offre.hasLinkServiceCivique,
+    [offre]
+  )
+
+  function redirectIfHasLink() {
+    if (hasLink) handleRedirectionOffre(offre)
+  }
+
   return (
-    <div
-      role='row'
-      className='table-row text-sm rounded-small shadow-s hover:bg-primary_lighten'
+    <tr
+      aria-label={hasLink ? "Ouvrir l'offre" : undefined}
+      onClick={redirectIfHasLink}
+      className={`text-base-regular rounded-small shadow-s ${
+        hasLink ? 'hover:bg-primary_lighten cursor-pointer' : ''
+      }`}
     >
-      <div role='cell' className='table-cell p-3 align-middle rounded-l-small'>
-        {offre.id}
-      </div>
-      <div role='cell' className='table-cell p-3 text-base-medium align-middle'>
-        {offre.titre}
-      </div>
-      <div role='cell' className='table-cell p-3 align-middle'>
-        {offre.organisation}
-      </div>
-      <div role='cell' className='table-cell p-3 align-middle rounded-r-small'>
+      <td className='p-3 align-middle rounded-l-small'>{offre.id}</td>
+      <td className='p-3 text-base-medium align-middle'>{offre.titre}</td>
```

Comment:
```suggestion
      <td className='p-3 text-base-bold align-middle'>{offre.titre}</td>
```
d'après la maquette, c'est en bold